edge-lambdas/src/viewer-response/viewer-response.ts

```diff
@@ -14,10 +19,15 @@ import {setHeader} from '../utils'
 export function getHandler(config: Config) {
     const handler: CloudFrontResponseHandler = async (event) => {
         let response = event.Records[0].cf.response
+        const request = event.Records[0].cf.request
+
+        const translationCursor = getHeader(request, TRANSLATION_CURSOR_HEADER) || 'default'
 
         response = addSecurityHeaders(response, config)
         response = addCacheHeader(response)
         response = addRobotsHeader(response, config)
+        response = addCookieHeaders(response, translationCursor)
+        response = addPreloadHeaders(response, request, translationCursor)
```

Comment:
For both of these methods, I think the name should have header in singular?
```suggestion
        response = addCookieHeader(response, translationCursor)
        response = addPreloadHeader(response, request, translationCursor)
```